epimodel/exports/epidemics_org.py

```diff
@@ -272,6 +276,8 @@ def get_df_else_none(df: pd.DataFrame, code) -> Optional[pd.DataFrame]:
     else:
         return None
 
+def get_df_list(df: pd.DataFrame, code) -> Optional[pd.DataFrame]:
+    return df.loc[df.index == code].sort_index()
```

Comment:
Why not reusing `get_df_else_none`? :thinking: Also, `df.loc[something]` is equivalent to `df.loc[df.index == something]`

Reply:
I think the difference is is that when there is only one `something` then `def.loc[something]` just returns the record (?) instead of a list of rows. This was nessesary because for the timezones there are multiple timezones per country.

Reply:
I don't think it returns just one:
![image](https://user-images.githubusercontent.com/2741256/79041237-772b9800-7bee-11ea-8f5f-a8948dfa7339.png)

Reply:
But my point is that if it is only one then it does not return a list of length one but just a single record

Reply:
![image](https://user-images.githubusercontent.com/2741256/79042419-0a1d0000-7bf8-11ea-9bd6-b13ea3aa6b06.png)
in that case you want to put `[ ]` around the identifier.

Reply:
Yeah but for the things it is expecting `None` or `object`.